charts/prefect-worker/values.yaml

```
@@ -55,6 +55,13 @@ worker:
     # -- additional sidecar containers
     extraContainers: []
 
+    # -- set dns policy and config for worker deployment
+    # dnsPolicy: ""
+    # dnsConfig:
+    #   nameservers: []
+    #   searches: []
+    #   options: []
```

```suggestion
  -- set dns policy and config for worker deployment
  dnsPolicy: ""
  -- <add note>
  dnsConfig:
    -- <add note>
    nameservers: []
    -- <add note>
    searches: []
    -- <add note>
    options: []
```

Small suggestion here:
- Outdent this section so it's under `worker` instead of `worker.initContainer`
- Un-comment the keys so they're included in the `helm-docs` generation of the README
- Add comments for each key so they have a description in the README